impeller/renderer/backend/gles/command_buffer_gles.cc

```diff
@@ -64,4 +64,13 @@ std::shared_ptr<BlitPass> CommandBufferGLES::OnCreateBlitPass() const {
   return pass;
 }
 
+// |CommandBuffer|
+std::shared_ptr<ComputePass> CommandBufferGLES::OnCreateComputePass() const {
+  if (!IsValid()) {
```

Comment:
Let's put the validity checks in the base class.

Reply:
Done